src/parser.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
         this.processed = proc;
     },
     processChange: function (file) {
+        if(file.endsWith('.csv')){
```

Comment:
A bit unnecessary since watcher.js already did that.

Reply:
Additionally, if you're going to do this - invert the condition and return early so you don't have a huge block of code now inside of an if-statement with limited other handling happening in the else block.